portfolio/src/main/webapp/index.html

```diff
@@ -10,54 +10,104 @@
+                </td>
             </tr>
             <tr>
-                <td bgcolor="#FFA500" id="aboutMe-info">A bilingual and extroverted Computer Science and Spanish double-major with exceptional communication, interpersonal and leadership skills. First-generation student who thrives in cultures that embrace diversity and reward imagination. Experience in software engineering, Android development and machine learning. </td>
+                <td id="name"><h1>David Vargas Puga</h1></td>
             </tr>
+            <tr id="socials-row">
+                    <td>
+                        <ul id="socials">
+                            <li id="github">
+                                <a href="https://github.com/davidvargaspuga"><img class="social-img" id="github-img" src="./images/github_logo_trans.png" /></a>
+                            <li>
+                            <li id="linkedin">
+                                <a href="https://www.linkedin.com/in/david-vargas-puga/"><img class="social-img" id="linkedin-img" src="./images/linkedin_logo.png" /></a>
+                            <li>
+                            <li id="insta">
+                                <a href="https://www.instagram.com/davidvargas.jpg/"><img class="social-img" id="insta-img" src="./images/insta_logo.png" /></a>
+                            <li>
+                        </ul>
+                    </td>
+                </tr>
             <tr>
-                <td id="socials-row">
-                    <ul id="socials">
-                        <li id="github">
-                            <a href="https://github.com/davidvargaspuga"><img id="github-img" src="./images/github_logo_trans.png" /></a>
-                        <li>
-                        <li id="linkedin">
-                            <a href="https://www.linkedin.com/in/david-vargas-puga/"><img id="linkedin-img" src="./images/linkedin_logo.png" /></a>
-                        <li>
-                    </ul>
-                </td>
+                <td id="aboutMe"><h2 class="section-titles">About Me</h2></td>
             </tr>
             <tr>
-                <td id="projects">Projects</td>
+                <td bgcolor="#FFA500" id="aboutMe-info">A bilingual and extroverted Computer Science and Spanish double-major at the University of Portland with exceptional communication, interpersonal and leadership skills. First-generation student who thrives in cultures that embrace diversity and reward imagination. Experience in software engineering, Android development and machine learning. </td>
             </tr>
             <tr>
-                <td>Project 1</td>
+                <td id="projects"><h2 class="section-titles">Projects</h2></td>
             </tr>
-            <tr>
-                <td>Project 2</td>
+            <tr class="unify-row">
+                <td id="unify-col">
+                    <p class="project-titles" id="unify-project-title">
```

Comment:
.project-titles is good use of CSS classes since this styling applies to all of the multiple instances of this. Good job!

Reply:
unify-project-title unnecessary